quickstart/Makefile

```
@@ -198,6 +208,14 @@ restart-backend: build-backend ## Build and start the application
 restart-frontend: build-frontend ## Build and start the application
 	$(call docker-compose, restart nginx)
 
+.PHONY: restart-service
+restart-service:
+	@if [ -z "${SERVICE}" ]; then \
+	  echo "Usage: make restart-service SERVICE=<service>"; exit 1; \
+	fi
+	$(call docker-compose, rm -s -f ${SERVICE})
+	$(call docker-compose, up -d --no-recreate)
```

Should this maybe be:

```suggestion
	$(call docker-compose, up -d --no-recreate ${SERVICE})
```

That is, bring up the service you `rm` d rather than all the services?